values.yaml

```diff
@@ -966,15 +997,13 @@ meshGateway:
   hostPort: null
 
   # Resource settings for mesh gateway pods.
-  # NOTE: The use of a YAML string is deprecated. Instead, set directly as a
-  #       YAML map.
   resources:
     requests:
-      memory: "128Mi"
-      cpu: "250m"
+      memory: "100Mi"
+      cpu: "100m"
```

Comment:
Since we're decreasing default resources, should this be called out as a breaking change too?

Reply:
good call! will add.